src/pages/GenderPage/GenderPage.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from "react";
+import { TextField } from "@material-ui/core";
+import "./GenderPage.css";
+import { useNavigate } from "react-router-dom";
+
+const GenderPage = () => {
+  const navigate = useNavigate();
+  const [gender, setGender] = useState("");
+  const isFormValid = !!gender;
```

Comment:
Here no need to use `!` twice. It's same as you not using it.
`!!gender` and `gender` will have same value.

Reply:
Removed **!!** from gender page.